### ISSUE TITLE
docs.rs build failing for v0.16.0

### ISSUE THREAD

**Issue by cmackenzie1** [delta-io/delta-rs#1714]
[docs.rs](https://docs.rs/crate/deltalake/latest)
[build logs](https://docs.rs/crate/deltalake/0.16.0/builds)

Likely from building with `--all-features` flag. 
```
[INFO] [stderr] error[E0252]: the name `tls` is defined multiple times
[INFO] [stderr]   --> /opt/rustwide/cargo-home/registry/src/index.crates.io-6f17d22bba15001f/rusoto_core-0.47.0/src/lib.rs:25:5
[INFO] [stderr]    |
[INFO] [stderr] 23 | use hyper_rustls as tls;
[INFO] [stderr]    |     ------------------- previous import of the module `tls` here
[INFO] [stderr] 24 | #[cfg(feature = "native-tls")]
[INFO] [stderr] 25 | use hyper_tls as tls;
[INFO] [stderr]    |     ^^^^^^^^^^^^^^^^ `tls` reimported here
[INFO] [stderr]    |
[INFO] [stderr]    = note: `tls` must be defined only once in the type namespace of this module
[INFO] [stderr] help: you can use `as` to change the binding name of the import
[INFO] [stderr]    |
[INFO] [stderr] 25 | use hyper_tls as other_tls;
[INFO] [stderr]    |     ~~~~~~~~~~~~~~~~~~~~~~
[INFO] [stderr] 
[INFO] [stderr]     Checking lz4 v1.24.0
[INFO] [stderr]     Checking blake2 v0.10.6
[INFO] [stderr]     Checking sha2 v0.10.8
[INFO] [stderr]     Checking itertools v0.11.0
[INFO] [stderr]     Checking zstd v0.12.4
[INFO] [stderr]     Checking fastrand v2.0.1
[INFO] [stderr] error[E0599]: no function or associated item named `new` found for struct `hyper_rustls::HttpsConnector` in the current scope
[INFO] [stderr]    --> /opt/rustwide/cargo-home/registry/src/index.crates.io-6f17d22bba15001f/rusoto_core-0.47.0/src/request.rs:224:41
[INFO] [stderr]     |
[INFO] [stderr] 224 |         let connector = HttpsConnector::new();
[INFO] [stderr]     |                                         ^^^ function or associated item not found in `HttpsConnector<_>`
[INFO] [stderr] 
[INFO] [stderr] error[E0599]: no function or associated item named `new` found for struct `hyper_rustls::HttpsConnector` in the current scope
[INFO] [stderr]    --> /opt/rustwide/cargo-home/registry/src/index.crates.io-6f17d22bba15001f/rusoto_core-0.47.0/src/request.rs:238:41
[INFO] [stderr]     |
[INFO] [stderr] 238 |         let connector = HttpsConnector::new();
[INFO] [stderr]     |                                         ^^^ function or associated item not found in `HttpsConnector<_>`
[INFO] [stderr] 
[INFO] [stderr] Some errors have detailed explanations: E0252, E0599.
[INFO] [stderr] For more information about an error, try `rustc --explain E0252`.
[INFO] [stderr] error: could not compile `rusoto_core` (lib) due to 3 previous errors
```

**Comment by rtyler** [comment]
This is already fixed #1693

**Comment by cmackenzie1** [comment]
Haha thanks, I only checked open issues before creating this 🫣
